applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc

```diff
@@ -0,0 +1,30 @@
+[id="configuring-custom-helm-chart-repositories"]
+
+= Configuring custom Helm chart repositories
+include::modules/common-attributes.adoc[]
+:context: configuring-custom-helm-chart-repositories
+
+toc::[]
+
+You can install Helm charts on an {product-title} cluster using the CLI. The *Developer Catalog*, in the *Developer* perspective of the web console, displays the Helm charts available in the cluster. By default, it lists the Helm charts from the Red Hat Helm chart repository. For a list of the charts see link:https://redhat-developer.github.io/redhat-helm-charts/index.yaml[the Red Hat `Helm index` file].
```

Comment:
To confirm, might be missing some context, but why aren't we linking users to the new `https://charts.openshift.io/index.yaml`?

Reply:
Hi @zonggen @Srivaralakshmi this link was given by the Helm developers to us when we first documented this. If the link needs to be changed we can do that, but can you double check this with David P and the team before we make the change?

Reply:
The new one https://charts.openshift.io/index.yaml, please